### DISCUSSION TITLE
Entity count placeholder

### DISCUSSION THREAD

**Discussion by Davisiiiik** [BG-Software-LLC/SuperiorSkyblock2#980] [Implemented Feature Requests] [answered]
So you have %superior_island_block_limit_<block>% and %superior_island_count_<block>% placeholders to show block limit and actual block count, but speaking of entities, you have only %superior_island_entity_limit_<entity>% describing the limit of them.

Is it possible to add placeholder like %superior_island_entity_count_<block>% to show current number of given entities?

**Comment by OmerBenGera** [comment, marked as answer]
It will be added next update.
